packages/manager/modules/emailpro/src/domain/spf-autoconfig/emailpro-domain-spf-autoconfig.html

```diff
@@ -0,0 +1,52 @@
+<div data-ng-controller="EmailProDomainSpfAutoconfigCtrl as ctrl">
+    <div
+        data-wizard
+        data-wizard-on-cancel="resetAction"
+        data-wizard-on-finish="configSPF"
+        data-wizard-title="'emailpro_tab_domain_diagnostic_spf_title' | translate"
+    >
+        <div data-wizard-step data-wizard-step-valid="getIsOVHDomain">
+            <div class="text-center" data-ng-if="ctrl.loading.step1">
+                <oui-spinner></oui-spinner>
+            </div>
+
+            <div data-ng-if="!ctrl.loading.step1">
+                <p
+                    data-ng-if="ctrl.domainDiag.isOvhDomain"
+                    data-translate="emailpro_tab_domain_diagnostic_spf_intro_ovh"
+                    data-translate-values="{ t0: ctrl.domain.displayName }"
+                ></p>
+                <p
+                    data-ng-if="!ctrl.domainDiag.isOvhDomain"
+                    data-translate="emailpro_tab_domain_diagnostic_spf_no_ovh"
+                ></p>
+                <dl class="dl-horizontal">
+                    <dt
+                        data-translate="emailpro_tab_domain_diagnostic_spf_label_champ"
+                    ></dt>
+                    <dd
+                        data-ng-bind-html="'emailpro_tab_domain_diagnostic_spf_config' | translate: { t0: ctrl.domainDiag.spf.subDomain, t1: ctrl.domainDiag.spf.priority, t2: ctrl.domainDiag.spf.weight, t3: ctrl.domainDiag.spf.port, t4: ctrl.domainDiag.spf.target }"
```

Comment:
if you have `"emailpro_tab_domain_diagnostic_spf_config": "Sous domaine : <strong>{0}</strong><br /> Cible : <strong>{4}</strong>",`
Why do you add `t1`, `t2` and `t3` as arguments here ?

Reply:
Moreover, keys seem to be wrong `{{t0}}` <> `{0}`. Here you're using args as object instead of array

Reply:
Hello @maximeVaillant . Can you just fixed this part for the PR
```suggestion
                        data-ng-bind-html="'emailpro_tab_domain_diagnostic_spf_config' | translate: { t0: ctrl.domainDiag.spf.subDomain, t1: ctrl.domainDiag.spf.target }"
```
and also update translate key by replacing `{0}` and `{4}` with `{{t0}}` and `{{t1}}`.
The rest is OK
Thank you